ast-transformations-core/src/main/kotlin/org/jetbrains/research/ml/ast/gumtree/diff/PsiElementTransformer.kt

```diff
@@ -116,13 +118,40 @@ class PsiElementTransformer(
         this.applyDelete(transformation)
     }
 
+    private fun String.replaceInRange(startOffset: Int, oldValue: String, newValue: String): String {
```

Comment:
I think you just invented https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/replace-range.html